doc/contributing/reviewing-contributions.chapter.md

```diff
@@ -302,6 +302,12 @@ Container system, boot system and library changes are some examples of the pull
 
 It is possible for community members that have enough knowledge and experience on a special topic to contribute by merging pull requests.
 
+In case the PR is stuck waiting for the original author to apply a trivial
+change (a typo, capitalisation change, etc.) and the author allowed the members
```

Comment:
Maybe this should also become a checkmark in the PR template?

Also, does this apply to non-members to recreate the PR under their own name, keep the original commits and make the changes in the recreated PR?

Reply:
> Maybe this should also become a checkmark in the PR template?

How would this look/why would it fit there? The checklist is for guidelines/tests that were run, I don't really see how it fits there.

> Also, does this apply to non-members to recreate the PR under their own name, keep the original commits and make the changes in the recreated PR?

I feel like this may be a separate issue to discuss, since pushing a fix to an existing PR than closing that PR and replacing it with the same changes + a trivial fix, IMO. Not sure, though.

Reply:
The checkmark would be for allowing changes by maintainers.